drivers/timers/delay.c

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/sparc/src/common/sparc_udelay.c
+ * drivers/timers/delay.c
```

Comment:
should we move to sched folder? e.g. sched/clock.

Reply:
I'm not sure, I'll let someone more experienced decide. I wasn't sure about putting it in the timer drivers either, but I guess this isn't really part of the scheduler either? I think it's closer to the alarm/timer drivers since they have similar functions.